idunn/api/urlsolver.py

```diff
@@ -39,7 +39,7 @@ async def follow_redirection(
     if not hmac.compare_digest(hash_url(url), hash):
         raise HTTPException(403, detail="provided hash does not match the URL")
 
-    response = await client.get(url, allow_redirects=False)
+    response = await client.get(url, follow_redirects=False)
```

Comment:
This parameter doesn't seem to have changed. I think this was a mistake, but the default value is actually `False` so it had no impact (still better to be explicit here). Either *pylint* go better or *httpx* has a better type annotation.

Cf. https://www.python-httpx.org/compatibility/#redirects